### ISSUE TITLE
Host and path missing from terminal's title bar

### ISSUE THREAD

**Issue by jhakonen** [andresgongora/synth-shell#142]
**Describe the bug**
Before I took the fancy prompt into use my Terminator terminal (and Gnome's default terminal too) had a username, host and current working directory in the terminal's title bar, like so:
![original-look](https://user-images.githubusercontent.com/1950698/70372581-8372b200-18e9-11ea-8706-87f6a83893c8.png)

The title bar seems to be set in .bashrc in following piece of code:
```bash
# If this is an xterm set the title to user@host:dir
case "$TERM" in
xterm*|rxvt*)
    PS1="\[\e]0;${debian_chroot:+($debian_chroot)}\u@\h: \w\a\]$PS1"
    ;;
*)
    ;;
esac
```

Now with the fancy prompt all I have is "jhakonen:", and it doesn't change no matter which directory I go in:
![path-missing](https://user-images.githubusercontent.com/1950698/70372668-c2553780-18ea-11ea-8e70-39572e3581b1.png)

**Expected behavior**
I changed this line https://github.com/andresgongora/synth-shell/blob/039e703/synth-shell/fancy-bash-prompt.sh#L144 from this:
```bash
PS1=$FBP_PS1
```
To this:
```bash
PS1="${FBP_PS1}\e]0;\u@\h: \w\a"
```
After that the window title looks like what it looked like originally:
![original-look-with-fancy](https://user-images.githubusercontent.com/1950698/70372737-b28a2300-18eb-11ea-9dc2-7fac02c6ecb5.png)

**Additional context**
I originally found about synth-shell from this youtube video: https://www.youtube.com/watch?v=iaXQdyHRL8M
The video's author didn't seem to have this issue with the title bar, in the video the path seems to show up correctly. On the other hand, it seems he used quite old version of synth-shell.

Is this a bug? Or is the title bar supposed to not have the host and path?

**Comment by andresgongora** [comment]
Hi! Thanks for your big report. I'm on the road right now, but should
return soon(ish).

It's definitely a bug, probably caused during some upgrade of the code.
I'll fix it and push to `develop` for now. During Christmas I want to push
all the changes we've done during the past months to `master` :)

Again, thanks for pointing the bug out!

**Comment by andresgongora** [comment]
Ok. I see the problem. It's caused by the order and when the variables are updated. This requires a bit of tweeking...

**Comment by andresgongora** [comment]
I'm working on this, but it requires a major overhaul of the script. This might take me some days, as I have no free time right now (we are closing projects before the holidays).

**Comment by jhakonen** [comment]
No worries, there is no hurry with this :) And it is completely understandable, I'm like wise burning midnight oil before holidays, and after work can't find much time or energy to do anything else.

**Comment by andresgongora** [comment]
Should be fixed in `develop` with bdb7e8e. Please let me know if the problem persists for you, but I'm quite positive the problem is solved :D

During the holidays I will merge all the latest fixes and improvements to `master` :)

**Comment by andresgongora** [comment]
Just pushed a quick fix to 844bb55 to also show the host name in the titlebar ;)

**Comment by jhakonen** [comment]
Just tested with `master`, works great!
